apps/pie-docs/src/assets/styles/_variables.scss

```diff
@@ -7,4 +7,12 @@ $hero-banner-header-wide-height: 600px;
     --header-height: 64px;
     --sidenav-width-narrow: 560px;
     --sidenav-width-wide: 260px;
+    --docsearch-highlight-color: none;
```

Comment:
As these are all related specifically to search, you could just include them at the top of that SCSS file? (inside a separate `:root { … }` block